docker/ingestd/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+source /opt/lsst/software/stack/loadLSST.bash
+setup -r /home/lsst/ctrl_ingestd
```

Comment:
If this is installed with `eups distrib install` instead of `git clone`, then I think this can be just `setup ctrl_ingestd`.  (Possibly you'll need to make sure the `current` tag is applied in eups.)

Reply:
ctrl_ingestd isn't part of distrib, so we can't do this.